src/forms/reset-password-popup/reset-password/data/hooks.js

```diff
@@ -1,25 +1,21 @@
-import { useEffect, useState } from 'react';
+import { useEffect } from 'react';
 
 /**
- * A react hook used to detect the authMode param in
+ * A react hook used to detect the password reset token from query param in
  * url and also remove it once its detected.
- * returns authMode param.
  */
-const useGetAuthModeParam = () => {
-  const [activationMessage, setActivationMessage] = useState(null);
+const useGetPasswordResetToken = () => {
+  const url = new URL(window.location.href);
+  const token = url.searchParams.get('password_reset_token', '');
 
   useEffect(() => {
-    const url = new URL(window.location.href);
-
-    const authModeParam = url.searchParams.get('authMode');
-    if (authModeParam) {
-      setActivationMessage(authModeParam);
-      url.searchParams.delete('authMode');
+    if (token && url.searchParams.get('password_reset_token', '')) {
+      url.searchParams.delete('password_reset_token');
```

Comment:
Should we delete param here or it should be on close dialog?

Reply:
on closing the dialog. updated.